source/BoardingPanel.cpp

```diff
@@ -536,17 +536,19 @@ void BoardingPanel::DoKeyboardNavigation(const SDL_Keycode key)
 
 // Constructor (commodity cargo).
 BoardingPanel::Plunder::Plunder(const string &commodity, int count, int unitValue)
-	: name(commodity), outfit(nullptr), count(count), unitValue(unitValue)
+	: name(commodity), outfit(nullptr), count(count), unitValue(unitValue), knownToPlayer(true)
 {
 	UpdateStrings();
 }
 
 
 
 // Constructor (outfit installed in the victim ship or transported as cargo).
-BoardingPanel::Plunder::Plunder(const Outfit *outfit, int count)
-	: name(outfit->DisplayName()), outfit(outfit), count(count),
-	unitValue(outfit->Cost() * (outfit->Get("installable") < 0. ? 1 : Depreciation::Full()))
+BoardingPanel::Plunder::Plunder(const PlayerInfo &player,
+								const Outfit *outfit, int count, bool outfitIsKnown)
+	: name(outfitIsKnown ? outfit->DisplayName() : "Unknown Outfit"), outfit(outfit), count(count),
+	unitValue((outfitIsKnown ? outfit->Cost() : outfit->Mass()) * (outfit->Get("installable") < 0. ? 1 : Depreciation::Full())),
```

Comment:
```suggestion
	unitValue((outfitIsKnown ? outfit->Cost() : 
		outfit->Mass()) * (outfit->Get("installable") < 0. ? 1 : Depreciation::Full())),
```

Reply:
I know, I have a fix committed however I'm waiting for further fixes

Reply:
fixed, thanks